sql-statements/sql-statement-calibrate-resource.md

```diff
@@ -18,21 +18,106 @@ The `CALIBRATE RESOURCE` statement is used to estimate and output the ['Request
 ## Synopsis
 
 ```ebnf+diagram
-CalibrateResourceStmt ::= 'CALIBRATE' 'RESOURCE'
+CalibrateResourceStmt ::= 'CALIBRATE' 'RESOURCE' WorkloadOption
+
+WorkloadOption ::=
+( 'WORKLOAD' ('TPCC' | 'OLTP_READ_WRITE' | 'OLTP_READ_ONLY' | 'OLTP_WRITE_ONLY') )
+| ( 'START_TIME' 'TIMESTAMP' ('DURATION' stringLit | 'END_TIME' 'TIMESTAMP')?)?
+
 ```
 
+## Privileges
+
+To execute this command, you need the following configuration and privileges:
+
+- You have enabled [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-new-in-v660).
+- You have `SUPER` or `RESOURCE_GROUP_ADMIN` privilege.
+- You have the `SELECT` privilege for all tables in the `METRICS_SCHEMA` schema.
+
+## Methods for estimating capacity 
+
+TiDB provides two methods for estimation:
+
+### Estimate capacity based on actual workload
+
+If your application is already running online, or you can run actual business tests, it is recommended to use the actual workload over a period of time to estimate the total capacity. Observe the following constraints to improve the accuracy of the estimation:
+
+- Use the `START_TIME` parameter to specify the time point at which the estimation starts, in the format of "2006-01-02 15:04:05". The default estimation end time is the current time.
+- After specifying the `START_TIME` parameter, you can use the `END_TIME` parameter to specify the estimation end time, or use the `DURATION` parameter to specify the estimation time window from `START_TIME`.
+- The time window ranges from 10 minutes to 24 hours.
+- In the specified time window, if the CPU utilization of TiDB and TiKV is too low, you cannot estimate the capacity.
+
+### Estimate capacity based on hardware deployment
+
+This approach is mainly based on the current cluster configuration, combined with the empirical values observed for different workloads for estimation. Because different types of workloads require different ratios of hardware, the output capacity of the same configuration of hardware might be different. The `WORKLOAD` parameter here accepts the following different workload types. The default value is `TPCC`.
```

Comment:
```suggestion
This method mainly estimates capacity based on the current cluster configuration, combined with the empirical values observed for different workloads. Because different types of workloads require different ratios of hardware, the output capacity of the same configuration of hardware might be different. The `WORKLOAD` parameter here accepts the following different workload types. The default value is `TPCC`.
```